albumentations/augmentations/functional.py

```diff
@@ -1514,7 +1514,7 @@ def fancy_pca(img, alpha=0.1):
 @clipped
 def glass_blur(img, sigma, max_delta, iterations, dxy, mode):
     coef = MAX_VALUES_BY_DTYPE[img.dtype]
-    x = np.uint8(cv2.GaussianBlur(np.array(img) / coef, sigmaX=sigma, ksize=(0, 0)) * coef)
+    x = np.uint8(cv2.GaussianBlur(np.array(img) / coef, sigmaX=sigma, ksize=(0, 0)) * 255)
```

Comment:
Better to simple remove conversion into `uint8` and convert to image data type
```python
x = cv2.GaussianBlur(np.array(img) / coef, sigmaX=sigma, ksize=(0, 0)) * coef
x = x.astype(img.dtype)
```

Reply:
@creafz @BloodAxe @ternaus Here is a little bit strange implementation. Maybe we will remove normalization before applying blur? After these changes transform will work fine, but the result may differ from this implementation by 2 `abs(old_result - new_result).max() <= 2`

Reply:
@Dipet, I agree with removing this normalization. I assume that it was used in the original implementation of glass_blur (https://github.com/hendrycks/robustness/blob/master/ImageNet-C/create_c/make_imagenet_c.py#L300) because skimage works mostly with float images that have values in the range [0.0, 1.0].